testsuite/tests/apicast/policy/conditional/test_batcher_policy_mapping_rules.py

```diff
@@ -0,0 +1,97 @@
+"""
+Test checks that batcher policy works with conditional policy.
+"""
+
+from time import sleep
+import pytest
+
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+from testsuite import TESTED_VERSION, rawobj  # noqa # pylint: disable=unused-import
+
+
+pytestmark = [
+    pytest.mark.nopersistence,
+    pytest.mark.skipif("TESTED_VERSION < Version('2.16')"),
+]
+
+
+@pytest.fixture(scope="module")
+def policy_settings():
+    """Set policy settings"""
+    return rawobj.PolicyConfig(
+        "conditional",
+        {
+            "condition": {
+                "operations": [
+                    {"left": "{{ uri }}", "left_type": "liquid", "op": "==", "right": "/get1", "right_type": "plain"}
+                ]
+            },
+            "policy_chain": [{"name": "3scale_batcher", "configuration": {"batch_report_seconds": 50}}],
```

Comment:
I would use global constant in the testfile for batch_report_seconds, which is reused in api_client and test.